stress_test.py

```diff
@@ -0,0 +1,490 @@
+    @abstractmethod
+    def get_infinite_gpu_run_count(self):
+        pass
+
+    @abstractmethod
+    def get_disk_write_count(self):
+        pass
+
+    @staticmethod
+    def _simple_run(cl):
+        run_command([cl, 'run', 'echo stress testing...'])
+
+    @staticmethod
+    def _create_and_output_worksheet(cl, name):
+        run_command([cl, 'new', name])
+        run_command([cl, 'print', name])
+
+
+class LightStressTestRunner(StressTestRunner):
+    def __init__(self, cl):
+        self._cl = cl
+        super().__init__(cl)
+
+    def get_large_file_size_gb(self):
+        return 1
+
+    def get_bundle_uploads_count(self):
+        return 1
+
+    def get_create_worksheets_count(self):
+        return 2
+
+    def get_parallel_runs_count(self):
+        return 1
+
+    def get_num_of_docker_rounds(self):
+        return 1
+
+    def should_test_infinite_memory(self):
+        return False
+
+    def should_test_infinite_disk(self):
+        return False
+
+    def should_test_infinite_gpu(self):
+        return False
+
+    def get_infinite_gpu_run_count(self):
+        return 1
+
+    def get_disk_write_count(self):
+        return 1
+
+
+class HeavyStressTestRunner(StressTestRunner):
+    def __init__(self, cl):
+        self._cl = cl
+        super().__init__(cl)
+
+    def get_large_file_size_gb(self):
+        return 20
+
+    def get_bundle_uploads_count(self):
+        return 2000
+
+    def get_create_worksheets_count(self):
+        return 2000
+
+    def get_parallel_runs_count(self):
+        return 1000
+
+    def get_num_of_docker_rounds(self):
+        return 1000
+
+    def should_test_infinite_memory(self):
+        return True
+
+    def should_test_infinite_disk(self):
+        return True
+
+    def should_test_infinite_gpu(self):
+        return True
+
+    def get_infinite_gpu_run_count(self):
+        return 1000
+
+    def get_disk_write_count(self):
+        return 1000
+
+
+class CustomStressTestRunner(StressTestRunner):
+    def __init__(self, cl, args):
+        self._cl = cl
+        self._args = args
+        super().__init__(cl)
+
+    def get_large_file_size_gb(self):
+        return args.large_file_size_gb
+
+    def get_bundle_uploads_count(self):
+        return args.bundle_upload_count
+
+    def get_create_worksheets_count(self):
+        return args.create_worksheet_count
+
+    def get_parallel_runs_count(self):
+        return args.parallel_runs_count
+
+    def get_num_of_docker_rounds(self):
+        return args.large_docker_runs_count
+
+    def should_test_infinite_memory(self):
+        return args.test_infinite_memory
+
+    def should_test_infinite_disk(self):
+        return args.test_infinite_disk
+
+    def should_test_infinite_gpu(self):
+        return args.test_infinite_gpu
+
+    def get_infinite_gpu_run_count(self):
+        return args.infinite_gpu_runs_count
+
+    def get_disk_write_count(self):
+        return args.large_disk_write_count
+
+
+def main():
+    if args.light:
```

Comment:
I think the default should be light, not heavy because the stress test could do damage.